src/tools/wasm-link.cc

```diff
@@ -305,29 +305,52 @@ static void write_table_section(Context* ctx,
 }
 
 static void write_export_section(Context* ctx) {
+  bool seenMemory = false, singleMemoryExport = false;
```

Comment:
local variable style is `seen_memory` and `single_memory_export`. Probably should document this somewhere :-)